src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -750,6 +766,45 @@ public function links()
     }
 
     /**
+     * Returns an Image object for the first node in the list.
+     *
+     * @param string $method The method for the image (get by default)
+     *
+     * @return Image An Image instance
+     *
+     * @throws \InvalidArgumentException If the current node list is empty
+     *
+     * @api
+     */
+    public function image($method = 'get')
+    {
+        if (!count($this)) {
+            throw new \InvalidArgumentException('The current node list is empty.');
+        }
+
+        $node = $this->getNode(0);
+
+        return new Image($node, $this->baseHref, $method);
+    }
+
+    /**
+     * Returns an array of Image objects for the nodes in the list.
+     *
+     * @return Image[] An array of Image instances
+     *
+     * @api
+     */
+    public function images()
+    {
+        $images = array();
+        foreach ($this as $node) {
+            $images[] = new Image($node, $this->baseHref, 'get');
```

Comment:
IMO, the method should not be passed as argument for Image. It does not make sense